### DISCUSSION TITLE
Host "Question" issues on github, not spectrum.

### DISCUSSION THREAD

**Discussion by dimaqq** [geldata/gel#1180] [Ideas]
Sadly EdgeDB spectrum.chat is indexed rather poorly by the search engines.

Meanwhile, GitHub issues are indexed very well :)

Thus, if I have a good question, then perhaps the answer to it would be of benefit to user users, it would be much better if the content was on github than spectrum.

(alternatively, it could be on stackoverflow)

**Comment by elprans** [comment]
We would generally like to keep github issues dedicated to bug reports and feature discussions.  Using StackOverflow for StackOverflow-like questions is fine, actually.  @1st1 what do you think?

**Comment by 1st1** [comment]
SO is definitely fine. I think it's also important to keep Spectrum as it's more "real time" communication with us.

**Comment by elprans** [comment]
Yes, the "How do I do x?" questions are definitely SO material, whereas more general discussions about ideas, design etc are better in the forum format.

**Comment by 1st1** [comment]
We seem to agree on that. What are you actually suggesting to do? There's no way to create a community on SO, it's up to people when they start asking questions about edgedb there.

**Comment by elprans** [comment]
We can update our New Issue gate to suggest SO for "How do I" questions.  Well, and create an "edgedb" tag on SO and subscribe to it.

**Comment by 1st1** [comment]
> Well, and create an "edgedb" tag on SO and subscribe to it.

I don't have enough reputation to do that. Also you create a tag by submitting a first question and adding a tag (you can add only if your reputation is higher than 1500).

> We can update our New Issue gate to suggest SO for "How do I" questions.

Until the "EdgeDB" tag is there and we can have a working slack integration to follow that tag I'd be -1 on this.

**Comment by dimaqq** [comment]
OK, let me post a few questions :)

**Comment by dimaqq** [comment]
First Q up, https://stackoverflow.com/questions/60067770/is-35k-rows-s-slow-or-fast-for-edgedb
The `edgedb` tag is created :)

**Comment by elprans** [comment]
Awesome, thanks!  Would you be able to share your dataset? I'd like to play with it to give an informed answer there :-)

**Comment by dimaqq** [comment]
So far I found out that macOS host → Docker container is responsible for half the runtime.
Specifically, netcat tops out at ~40MB/s 😅 
https://dima.egnyte.com/dl/gUq6FH4qBq/users.json_

**Comment by 1st1** [comment]
@dimaqq We now have GitHub Discussions enabled and we'll be deprecating Spectrum. Consolidated at last!
